spec/support/playbook/rspec.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "pry"
```

Comment:
You will have to remove this too. It will likely fail your CI (or I hope it would).

Reply:
It did fail, and I thought we ran `git co` on this file. Good catch!